### ISSUE TITLE
React app does not load on publish instance due to HierarchyPageImpl

### ISSUE THREAD

**Issue by cqsapient** [adobe/aem-spa-project-archetype#87]
When there is no access to /conf folder ( and hence the page policy) to anonymous user on publish instance, then content attribute is empty as it gets populated via HierarchyPageImpl and code is not able to access conf folder

<meta property="cq:pagemodel_root_url"/>

so to fetch the model json it accesses the current page's model

 
![image](https://user-images.githubusercontent.com/22097490/59759625-dde46b80-92d3-11e9-9256-c5d7d7bfee56.png)

But when anonymous user has access to 'conf' folder, then it is populated

<meta property="cq:pagemodel_root_url" content="/content/mysite/en.model.json"/>  

and now there are two calls to for model.json

![image](https://user-images.githubusercontent.com/22097490/59759851-4fbcb500-92d4-11e9-91a5-2a0212b8d34d.png)

because of these two calls nothing gets loaded in the app. I know the call is originated by page model manager, but the app should load in case of two calls as well, or the page specific call should not be initiated when rootmodel url is populated.

For now I removed the content attribute in rootmodel meta tag altogether, please let me know the better and appropriate solution.

**Comment by cqsapient** [comment]
The two calls originate only on dispatcher - on publish, when cq:pagemodel_root_url is populated, only once call(rightly so) on the root page is sent to fetch the model, but in dispatcher with the shortened url and without .html , two calls originate. Please let me know if a cleaner solution is available.

**Comment by pfauchere** [comment]
Would giving read access to the project templates and policies to anonymous solve your issue?

**Comment by samuelmeuli** [comment]
This is the same issue as https://github.com/adobe/aem-spa-project-archetype/issues/75, and it should be fixed by https://github.com/adobe/aem-spa-project-archetype/pull/131.

Please open a new issue if this isn't resolved for you in the next version :)
